webapp-django/crashstats/settings/test.py

```diff
@@ -72,3 +72,9 @@
 
 MIDDLEWARE_RETRIES = 10
 MIDDLEWARE_MIDDLEWARE_RETRY_SLEEPTIME = 3
+
+# During testing, if mocking isn't done right, we never want to
+# accidentally send data to Google Analytics.
+GOOGLE_ANALYTICS_API_URL = 'https://fake.guugle-analytics.fake/collect'
```

Comment:
It's always better to use "example.com" for fake domain names. That's what it's for.